frontend-utils/src/parse.rs

```diff
@@ -146,19 +147,22 @@ pub trait ReadExt<'a> {
 
             cx.pop_key();
         }
+
+        result
```

Comment:
very, very, very small nit: this has a empty line before result unlike `get_array_of_tables`.